.github/workflows/gitleaks.yml

```diff
@@ -1,6 +1,6 @@
 name: gitleaks
 
-on: [push,pull_request]
```

Comment:
Not required for this PR:

A user is still able to commit sensitive information, because this is only ran on push, and on on commit. Would on pre-commit be a better place?

Reply:
This check is actually completely broken and doesn't work correctly.  It also takes about 27 minutes to run.  So yeah, this needs to be improved...